src/utils/aotf.js

```diff
@@ -848,15 +856,14 @@ async function _mutateError (mutationName, message, response) {
   }
 
   // open a user alert
-  await store.dispatch('setAlert', new AlertModel(
-    `command failed: ${mutationName} - ${message}`,
-    null,
-    'error')
+  await store.dispatch(
+    'setAlert',
+    new Alert(`Command failed: ${mutationName} - ${message}`, 'error')
   )
 
   // format a response
   return {
-    status: TaskState.SUBMIT_FAILED,
+    status: mutationStatus.FAILED,
```

Comment:
FYI: submit-failed makes more sense for most commands (e.g. broadcast) because an error response means that the scheduler didn't receive the command at all.

It's different for the UIS mutations (e.g. play) because they actually run the command synchronously so can return the runtime error.

We could probably do with being able to differentiate these one day but that will have to wait for - https://github.com/cylc/cylc-flow/issues/3329